src/blockchain/aeternity/config.ts

```diff
@@ -1,19 +1,38 @@
 import { Config } from '@jelly-swap/aeternity';
-import AppConfig from '../../../config';
+import UserConfig from '../../config';
+import { safeAccess } from '../../utils';
+import { logError } from '../../logger';
 
-export default {
-    ...Config(7200),
-    providerUrl: 'https://sdk-mainnet.aepps.com/',
-    internalUrl: 'https://sdk-mainnet.aepps.com/',
-    compilerUrl: 'https://compiler.aepps.com',
-    wsUrl: 'wss://mainnet.aeternal.io/websocket',
-    apiUrl: 'https://mainnet.aeternal.io/',
-    contractAddress: 'ct_jmRkfpzmn7KZbXbkEL9wueJkb1vaFzMnVFJMFjAnJgj1CTtQe',
-    explorer: 'https://explorer.aepps.com/transactions/',
-    REFUND_PERIOD: 10,
-    //Expiration is in milliseconds
-    VALID_EXPIRATION: 72000000,
+export default () => {
+    const userConfig = new UserConfig().getUserConfig();
 
-    receiverAddress: AppConfig.BLOCKCHAIN.AE.ADDRESS,
-    KEY_PAIR: AppConfig.BLOCKCHAIN.AE.SECRET,
+    const address = safeAccess(userConfig, ['WALLETS', 'AE', 'ADDRESS']);
+    const secret = safeAccess(userConfig, ['WALLETS', 'AE', 'SECRET']);
+
+    const config = {
+        ...Config(7200),
+        providerUrl: 'https://sdk-mainnet.aepps.com/',
+        internalUrl: 'https://sdk-mainnet.aepps.com/',
+        compilerUrl: 'https://compiler.aepps.com',
+        wsUrl: 'wss://mainnet.aeternal.io/websocket',
+        apiUrl: 'https://mainnet.aeternal.io/',
+        contractAddress: 'ct_jmRkfpzmn7KZbXbkEL9wueJkb1vaFzMnVFJMFjAnJgj1CTtQe',
+        explorer: 'https://explorer.aepps.com/transactions/',
+        REFUND_PERIOD: 10,
+        //Expiration is in milliseconds
+        VALID_EXPIRATION: 72000000,
+    };
+
+    if (address && secret) {
+        return {
+            ...config,
+            receiverAddress: address,
+            KEY_PAIR: {
+                publicKey: userConfig.WALLETS['AE'].ADDRESS,
```

Comment:
You can use address and secret  rather than userConfig.Wallet['AE'].Address.
You have done this already on line 9 and 10